src/MaterialUI.re

```diff
@@ -477,6 +477,69 @@ module Divider = {
     );
 };
 
+module Drawer = {
+  module Anchor = {
+    type t =
+      | Left
+      | Top
+      | Right
+      | Bottom;
+    let to_string =
+      fun
+      | Left => "left"
+      | Top => "top"
+      | Right => "right"
+      | Bottom => "bottom";
+  };
+  module Type = {
+    type t =
+      | Permanent
+      | Persistent
+      | Temporary;
+    let to_string =
+      fun
+      | Permanent => "permanent"
+      | Persistent => "persistent"
+      | Temporary => "temporary";
+  };
+  [@bs.module "material-ui/Drawer"] external reactClass : ReasonReact.reactClass = "default";
+  let make =
+      (
+        ~modalProps: option(Js.t({..}))=?,
+        ~slideProps: option(Js.t({..}))=?,
+        ~anchor: option(Anchor.t)=?,
+        ~classes: option(Js.t({..}))=?,
+        ~style: option(ReactDOMRe.style)=?,
+        ~elevation: option(int)=?,
+        ~onRequestClose: option((ReactEventRe.Mouse.t => unit))=?,
```

Comment:
not sure about this

Reply:
It might be `option(ReasonReact.Callback.t(unit))=?`, but would have to double check